### PR TITLE
Synced live releases shim

### PR DESCRIPTION
**Pull request by kitallis**
A potential way in where we can show two sets of release steps for two platforms in one single live release page. Many of these positional changes can be incorporated back if we prefer them.

Screenshot 2023-06-07 at 6 53 21 PM

Screenshot 2023-06-07 at 6 53 31 PM

Screenshot 2023-06-07 at 6 53 51 PM

### CODE REVIEW

File: app/views/shared/demo/live_release/_ios_stability.html.erb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<% release =
```
Comment (kitallis) [inline app/views/shared/demo/live_release/_ios_stability.html.erb:1]: This is a new file. The rest of the `demo/live_release` pages are copies.

File: app/views/trains/_form.html.erb [hunk 2]
```diff
@@ -114,7 +114,9 @@
 
   <%= render partial: "shared/demo/release_schedule_form", locals: { form: } %>
 
-  <div class="text-xl text-slate-600 font-medium mt-8 mb-4">Branching strategy</div>
+  <div class="border-t border-dashed border-slate-200 pt-8 mt-8"></div>
```
Comment (kitallis) [inline app/views/trains/_form.html.erb:117]: Minor unrelated change.

File: app/views/shared/_event_timeline.html.erb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+                <%= time_format(passport.event_timestamp) %>
+              </span>
+          </div>
+
+          <span class="text-xs inline-flex font-medium <%= passport_badge(passport) %> rounded-full text-center mr-3 ml-2 px-2 py-1">
+              <%= image_tag("#{passport_icon(passport)}.svg", width: 20, class: "inline-flex") %>
+            </span>
+
+          <div class="self-center text-slate-800 <%= short_width ? "truncate" : nil %>">
+            <%= raw passport.message %>
```
Comment (nid90) [inline app/views/shared/_event_timeline.html.erb:17]: are we not truncating these messages?

**Review by nid90 — APPROVED** [review]
